tests/examples/tensorflow/test_tf_model.py

```diff
@@ -12,29 +12,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Unit tests demo TF model."""
+import argparse
+import os
+import shutil
+import sys
+import tempfile
 import unittest
 
-from examples.tf import tf_model
+import tensorflow.compat.v1 as tf
 
+from tests import test_utils
 
-class TestModel(unittest.TestCase):
+
+class TestModel(tf.test.TestCase):
     """Tests TF demo model."""
 
+    @classmethod
+    def setUpClass(cls):
+        super(TestModel, cls).setUpClass()
+        cls.test_dir = tempfile.mkdtemp()
+        cls.demo_dir = os.path.join(cls.test_dir, 'demo')
+        shutil.copytree('examples/tf', cls.demo_dir)
+
+        # TODO(humichael) We can't import the model using __import__ because
```

Comment:
This is important to note. We can work on this, in the following updates when we have a proper test suite.

Reply:
agreed